installation/index.md

```diff
@@ -36,13 +36,16 @@ The previously mentioned [openHABian](openhabian.html) can also be used to kicks
 
 ## Prerequisites
 
-Make sure that you have an up to date version of the [Oracle Java Runtime Environment](https://java.com/en/download/manual.jsp) on your host system installed.
-A famous choice is OpenJDK, however, Oracle Java Version 8 is recommended for openHAB.
-openHAB may be unstable and slower executed by OpenJDK.
+Make sure that you have an up to date Java Runtime Environment (JRE) installed on your host system. The [Oracle JRE](https://java.com/en/download/manual.jsp)
```

Comment:
> @kaikreuzer any news?

Unfortunately not yet.

> So I would do the manual installation from tar for now and sneak in the repository whenever it is available. How does that sound for you?

If the tar installation is straight forward, this sounds like a plan 👍